src/os/bluestore/BlueStore.cc

```diff
@@ -5435,9 +5447,21 @@ void BlueStore::_close_path()
   path_fd = -1;
 }
 
-int BlueStore::_write_bdev_label(CephContext *cct,
-				 const string &path, bluestore_bdev_label_t label)
-{
+void BlueStore::_unalloc_bdev_label_offset(uint64_t offset) {
+  ceph_assert(alloc);
+  bool existing_allocation_in_offset = true;
+  alloc->foreach([&](uint64_t allocation_offset, uint64_t allocation_length) {
```

Comment:
Naming is just bad.
allocation_offset = means free region starts here
allocation_length = free region has this size